rpm/argparse-manpage.spec.tpl

```diff
@@ -1,4 +1,5 @@
-%if 0%{?fedora}
+# EPEL 9 (not EL 9 itself!) supports %%pyproject macros
```

Comment:
RHEL 9 supports %pyproject macros as well.

Reply:
With mock -r rhel-9-x86_64:
```
Executing(%generate_buildrequires): /bin/sh -e /var/tmp/rpm-tmp.qlPTXJ
+ umask 022
+ cd /builddir/build/BUILD
+ cd argparse-manpage-3
+ %pyproject_buildrequires
/var/tmp/rpm-tmp.qlPTXJ: line 32: fg: no job control
```

Reply:
You need to add BuildRequires: pyproject-rpm-macros